modules/express/src/config.ts

```diff
@@ -3,6 +3,19 @@ import { isNil, isNumber } from 'lodash';
 
 import { args } from './args';
 
+function readEnvVar(name, ...deprecatedAliases): string | undefined {
+  if (process.env[name] !== undefined) {
+    return process.env[name];
```

Comment:
won't this always return?
when do we ever call `readEnvVar(name)` such that `process.env[name]` in `undefined` ?

Reply:
If a user upgrades but does not change the environment variables they are setting, the one referred to by `name` might not be there (since that was not a valid option when they did their initial setup of the env vars). In that case `process.env[name]` would be undefined, and we would try the deprecated aliases for the variable instead. If they were using an older form of the env var, it should get picked up in the loop.

To make it more concrete, imagine a user who has been using express behind a reverse proxy for a while, and therefore sets `export DISABLESSL=true`. When they update, the new version will look at `BITGO_DISABLE_SSL` instead, and will find it to be undefined. In order to keep this user working, we'd need to look at the older forms as well. That's what the 2nd loop does.

Reply:
oh i see
makes sense!